app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MediaRanker</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+  <header class = "app-header__header">
+    <h1>
+      <a href="/" >Media Ranker</a> <small>  Ranking the Best of Everything</small>
+    </h1>
+    <nav class ="app-main__nav">
+      <ul class="nav app-header__site-nav-container">
+        <li class="nav-item app-header__nav_item">
+          <%= link_to "View top Media", root_path, class:"nav-link" %>
+        </li>
+        <li class="nav-item app-header__nav_item">
+          <%= link_to "View all Media", works_path, class:"nav-link" %>
+        </li>
+        <li class="nav-item app-header__nav_item">
+          <%= link_to "Add a new work", new_work_path, class:"nav-link" %>
+        </li>
+        <li class="nav-item app-header__nav_item">
+          <%= link_to "View all users", users_path, class:"nav-link"%>
+        </li>
+      </ul>
+
+        <ul class = "nav app-header__user-nav-container">
+          <% if session[:user_id] %>
+          <li class = "nav-item app-header__nav_item">
+            <%= button_to "Logout", logout_path, method: :post, class: "btn btn-primary" %>
+          </li>
+        <%else %>
+            <li class = "nav-item app-header__nav_item">
+          <%=button_to "Log in", login_path, method: :get %>
+            </li>
+        <%end %>
```

Comment:
This is something I would replace with an `@user` using a controller filter to find the current user.  That way the view doesn't have to know anything about session.